component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateAuthenticator.java

```diff
@@ -175,6 +170,42 @@ protected void processAuthenticationResponse(HttpServletRequest httpServletReque
         }
     }
 
+    /**
+     * @param userName              certificate's username
+     * @param authenticationContext the authentication context
+     * @param data                  certificate's data
+     * @param claims                claims of the user
+     * @param cert                  X509 certificate
+     * @throws AuthenticationFailedException
+     */
+    private void addOrValidateCertificate(String userName, AuthenticationContext authenticationContext, byte[] data,
```

Comment:
Its better to add the doc comment to describe the method.

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-x509/pull/15/commits/cf04c4989bb2c91bd5f2058d43c779f7cb46e9a1